integrations/client/clientset/versioned/typed/flux.weave.works/v1beta1/helmrelease.go

```diff
@@ -117,6 +118,22 @@ func (c *helmReleases) Update(helmRelease *v1beta1.HelmRelease) (result *v1beta1
 	return
 }
 
+// UpdateStatus was generated because the type contains a Status member.
+// Add a +genclient:noStatus comment above the type to avoid generating UpdateStatus().
```

Comment:
Same here

Reply:
Both results of running `./bin/update-codegen.sh`.

Reply:
@2opremio everything inside the `client` package is generated by the Kubernetes CRD codegen.

Reply:
thanks, I didn't know